modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -752,91 +693,64 @@ then the `--password-file` option will take precedence.
 
 === Optimizing wnd:process-queue
 
-NOTE: Do not use this option if the process-queue is fast enough. The option has some drawbacks,
-specifically regarding password changes in the backend.
+NOTE: Do not use this option if the process-queue is fast enough. The option has some drawbacks, specifically regarding password changes in the backend.
+
+`wnd:process-queue` creates all the storages that need to be updated from scratch. To do so, we need to fetch all the users from all the backends (currently only the ones that have logged in at least once because the others won't have the storages that we'll need updates).
```

Comment:
```suggestion
`wnd:process-queue` creates all the storages that need to be updated from scratch. To do so, we need to fetch all the users from all the backends (currently only the ones that have logged in at least once because the others won't have the storages that will need updates).
```